deny.toml

```diff
@@ -0,0 +1,13 @@
+[licenses]
+unlicensed = "deny"
+# We want really high confidence when inferring licenses from text
+confidence-threshold = 0.92
+allow = [
```

Comment:
Why this particular list?

Reply:
I copied mostly from tonic but should include a list of licenses we accept.

Reply:
This can be expanded though.

Reply:
If it helps, here is flurry's [`deny.toml`](https://github.com/jonhoo/flurry/blob/master/deny.toml).